src/node/recursiveModuleCompile.js

```diff
@@ -50,6 +50,8 @@ function recursiveModuleCompileToSingleFile(outputFile, includes, options) {
     return include;
   });
 
+  options.rootModules = resolvedIncludes;
```

Comment:
Maybe via metadata but it will need to be passed to the transformer too. There is a fair amount of complex workflow to get there..